scripts/start_cr_server.sh

```diff
@@ -2,76 +2,116 @@
 # (c) Crowdrender 2019 - 2022
 
 function download_cr {
-    if test -z $token; then
+
+
+    # exit early if a docker bind mount is specified, but no version number is provided.
+    if [ -z "$local_cr_path" ] && [ -z "$cr_version" ]; then
+        echo "ERROR: specifying local_cr_path requires you to also specify the version by setting the cr_version option."
+        exit 1
+
+    # if the token is not provided and neither has a local bind mount, then exit early.
+    elif [ -z "$local_cr_path" ] && [ -z $token ]; then
+
         echo "ERROR: you need to specify a login token in order to download the latest version!"
         exit 1
-    else 
-        echo "download token given with length of ${#token}"
+
     fi
 
-    # use the token for downloading
-    if [ $CR_VERSION == "latest" ]; then
+    if [ -z "$cr_version" ]; then
+        echo "No specific version set, defaulting to 'latest'."
+        cr_version="latest"
+    fi
+
+    # if the user provides a local path 
+    if ! [ -z "$local_cr_path" ]; then
+        
+        # use the provided path as the source of the addon package instead of downloading it
+        echo "Going to use $local_cr_path as the source of the addon package."
+
+    # alternatively, if the request version is the latest one...
+    elif [ $cr_version == "latest" ]; then
+
+        # use the token for downloading the addon package useing the dedicated 'latest' URL
         echo "downloading latest CR version!"
-        curl -# -H "Authorization: Bearer $token" -X GET https://discovery.crowd-render.com/api/download/addon/$CR_VERSION -o $CR_VERSION.zip
+        curl -# -H "Authorization: Bearer $token" -X GET https://discovery.crowd-render.com/api/download/addon/$cr_version -o $cr_version.zip
+    
+    # otherwise, use the provided version number.
     else
-        echo "downloading CR version: $CR_VERSION!"
-        curl -# -H "Authorization: Bearer $token" -X GET https://discovery.crowd-render.com/api/download/addon/$CR_VERSION.zip -o $CR_VERSION.zip
+        # 
+        echo "downloading CR version: $cr_version!"
+        curl -# -H "Authorization: Bearer $token" -X GET https://discovery.crowd-render.com/api/download/addon/$cr_version.zip -o $cr_version.zip
     fi
 }
 
 function setup_cr {
+    
+    # check to see if the local dev path is set, don't install the
+    # addon from discovery if so
+    if ! [ -z "$local_cr_path" ];
+    then
+        echo "using mounted location for CR addon"
+        /usr/local/blender/blender -b -noaudio --python ./install_addon.py -- -e
```

Comment:
in case you accept my suggestion in `install_addon.py`
```suggestion
        /usr/local/blender/blender -b -noaudio --python ./install_addon.py
```

Reply:
accepted!